cmd/performance-profile-creator/cmd/root.go

```diff
@@ -52,25 +59,22 @@ var rootCmd = &cobra.Command{
 	Short: "A tool that automates creation of Performance Profiles",
 	PreRunE: func(cmd *cobra.Command, args []string) error {
 		mcpName := cmd.Flag("mcp-name").Value.String()
-		kubeConfig := cmd.Flag("kube-config").Value.String()
-		clientset := NewClientSet(kubeConfig)
-		mcp, err := getMCPByName(mcpName, clientset)
+		mustGatherDirPath := cmd.Flag("must-gather-dir-path").Value.String()
```

Comment:
if the preferred way is to read from must-gather, could you please squash this commit with the previous one in the next upload? No point in adding code and removing it in the very next commit.

Reply:
Sure, will do